src/styles/main.scss

```diff
@@ -197,6 +140,7 @@ img {
     position: relative;
     padding-bottom: 10px;
     border-top: 1px solid #fff;
+    
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/matfilipowicz/BooGi/pullRequest?prid=5801182)